modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -106,6 +106,14 @@ The Ingress Operator also converts the TLS `1.0` of an `Old` or `Custom` profile
 Ciphers and the minimum TLS version of the configured security profile are reflected in the `TLSProfile` status.
 ====
 
+|`clientTLS`
+|`clientTLS` authenticates client access to the cluster and services using mutual TLS authentication. If not set, then client TLS is not enabled.
+
+`clientTLS` has the required subfields, `spec.clientTLS.clientCertificatePolicy` and `spec.clientTLS.ClientCA`.
+
+The `ClientCertificatePolicy` subfield accepts either one of two values: `Required` or `Optional`. The `ClientCA` subfield accepts a path to a ConfigMap, which is in the same namespace as the Ingress Controller, and an optional
```

Comment:
clientCA specifies a configmap.... The administrator must create this configmap in the openshift-config namespace. It is not in the same namespace as the Ingress Controller.

see https://github.com/openshift/api/blob/67c28690af52a69e0b8fa565916fe1b9b7f52f10/operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml#L68

Reply:
I updated the wording here